account_index_based_currency/models/res_company.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+
+class Company(models.Model):
+    _inherit = "res.company"
+
+    index_based_currency_id = fields.Many2one(
+        'res.currency',
+        help="Currency used por reporting purposes",
+        default=lambda self: self.env.ref('base.USD'))
```

Comment:
Why USD by default?

Reply:
Because is the most common rated used for index based currency.